api/proto/teleport/devicetrust/v1/device_collected_data.proto

```diff
@@ -81,4 +82,16 @@ message DeviceCollectedData {
   // The serial number of the "base board" as reported by BIOS DMI Type 2.
   // This field can be empty if no value has been configured.
   string base_board_serial_number = 13;
+
+  // If during the collection of this device data, the device performed a TPM
+  // platform attestation (e.g during enrollment or authentication), then this
+  // field holds the record of this attestation. This allows the state of the
+  // device to be compared to historical state, and allows for the platform
+  // attestations to be revalidated at a later date.
+  //
+  // This field is not explicitly sent up by the client, and any DCD sent by a
+  // client including this field should be rejected. The server should inject
+  // this field once verifying that the submitted platform attestation during
+  // the enrollment or authentication.
```

Comment:
Seems fair to say it's a "system managed" field, semantics are similar to others like it.

```suggestion
  // the enrollment or authentication.
  //
  // System managed.
```

Reply:
Yup, `system managed` seems nicer.

Reply:
627a5a8b17